extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -263,6 +266,34 @@ const TEMPLATE = {
         'class': 'i-amphtml-story-system-layer-buttons-start-position',
       }),
     },
+    {
+      tag: 'a',
```

Comment:
Do you need to do something to make it work with the AMP Linker?

Reply:
Good call, added support by propagating the click in light DOM using the helper Matias wrote.